test/jit_utils.py

```diff
@@ -71,44 +71,17 @@ def _isHookExceptionOk(self, e):
                 return True
         return False
 
-    def emitFunctionHook(self, func):
-        # func has invalid names for export, skip the jitter check
-        if func.name == "<lambda>" or "aten::" in func.name or not _inline_everything:
-            return
-        # disable the hook while we parse code, otherwise we will re-enter the hook
-        with torch.jit._disable_emit_hooks():
-            try:
-                src, constants = _jit_python_print(func)
-                cu = torch.jit.CompilationUnit()._import(src, constants)
-                func2 = getattr(cu, func.name)
-                src2, constants2 = _jit_python_print(func2)
-                self.assertMultiLineEqual(src, src2)
-            except RuntimeError as e:
-                if not self._isHookExceptionOk(e):
-                    raise
-
-    def emitModuleHook(self, module):
+    def _compared_saved_loaded(self, m):
```

Comment:
This will eventually have to be split apart again because saving a function will look different than saving a module with "new" serialization.